### DISCUSSION TITLE
Which of the following methods would you prefer?

### DISCUSSION THREAD

**Discussion by bitbound** [immense/Remotely#278] [Q&A]
With the new server installer, it will build from the source code available in your fork of Remotely.  That means you have to keep your fork updated with mine to build the latest versions. 

But it looks like I could also set it so it pulls from my repo. That means you wouldn't have to pull changes into your fork, and the latest changes would still be available. But it might make it more difficult to build from your repo if you've made customizations to the code base.

I could also make it an argument for the installer, but it's already getting a little bloated.

I'm curious if anyone has a preference? Let me know. :)

**Comment by swilkey** [comment]
Hi Jared
My preference would be as an argument.
I suspect that the majority of users would not be modifying the code base,
however we should still encourage those who want to because that can result
in improvements for all.
Stephen

**Comment by houstonbofh** [comment]
I agree that it should be possible.  Being a little difficult is not a bad thing.  Weeds out the people you may not want to support.  Perhaps a host file modification instead...
